localstack-core/localstack/services/lambda_/invocation/models.py

```diff
@@ -16,5 +17,8 @@ class LambdaStore(BaseStore):
     # maps layer names to Layers
     layers: dict[str, Layer] = LocalAttribute(default=dict)
 
+    # maps resource ARNs for EventSourceMappings and CodeSigningConfiguration to tags
```

Comment:
This should include Lambda Functions but I'm not sure it's worth re-triggering the workflow for. Also tags are currently doubly stored both in a Lambda `Function` objects `tag` field as well as in the `TaggingService` -- with the former likely being removed in a later release.

Reply:
Thanks for unifying the double storage 👏